contracts/connector_permissions/tests/connector_permissions_tests.rs

```diff
@@ -213,6 +216,7 @@ mod connector_permissions {
                 }))
                 .unwrap()
                 .gas(parse_gas!("300 Tgas") as u64)
+                .deposit(parse_near!("1"))
```

Comment:
If it is not too hard, also add under connector_permissions_tests a failing test due to too little deposit provided